tests/zeromq.terminal/PubSubTests/PubSubScenarios.cs

```diff
@@ -30,6 +30,7 @@ internal static async Task SimplePubSub(SocketConfiguration configuration)
             socket.RegisterSubscriber<PubSubMessage>(msg => System.Console.WriteLine("msg received: " + msg.Message));
             await socket.PublishAsync(new PubSubMessage { Message = "Published simply"});
 
+            System.Console.WriteLine("press any enter to exit");
```

Comment:
```suggestion
            System.Console.WriteLine("press any key to exit");
```

Reply:
But it is not any key on ReadLine is it? So the any needs to go, not the key.

Reply:
ah - yes - `Console.ReadLine` ... my bad